src/main/kotlin/io/github/anki/anki/service/secure/jwt/AuthEntryPointJwt.kt

```diff
@@ -1,28 +1,21 @@
 package io.github.anki.anki.service.secure.jwt
 
-import jakarta.servlet.ServletException
-import jakarta.servlet.http.HttpServletRequest
-import jakarta.servlet.http.HttpServletResponse
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.server.reactive.ServerHttpResponse
 import org.springframework.security.core.AuthenticationException
-import org.springframework.security.web.AuthenticationEntryPoint
+import org.springframework.security.web.server.ServerAuthenticationEntryPoint
 import org.springframework.stereotype.Component
-import java.io.IOException
+import org.springframework.web.server.ServerWebExchange
+import reactor.core.publisher.Mono
 
 @Component
-class AuthEntryPointJwt : AuthenticationEntryPoint {
-    @Throws(IOException::class, ServletException::class)
-    override fun commence(
-        request: HttpServletRequest,
-        response: HttpServletResponse,
-        authException: AuthenticationException,
-    ) {
-        LOG.error("Unauthorized error: {}", authException.message)
-        response.sendError(HttpServletResponse.SC_UNAUTHORIZED, "Error: Unauthorized")
-    }
+class AuthEntryPointJwt : ServerAuthenticationEntryPoint {
 
-    companion object {
-        private val LOG: Logger = LoggerFactory.getLogger(AuthEntryPointJwt::class.java)
+    override fun commence(exchange: ServerWebExchange, ex: AuthenticationException): Mono<Void> {
+        val response: ServerHttpResponse = exchange.response
+        response.setStatusCode(HttpStatus.UNAUTHORIZED)
+        return response.writeWith(
+            Mono.just(response.bufferFactory().wrap("Error: Unauthorized1".encodeToByteArray())),
```

Comment:
```suggestion
            Mono.just(response.bufferFactory().wrap("Error: Unauthorized".encodeToByteArray())),
```

Reply:
yeah, forgot to delete when i tried to break tests